app/protocols/ssb-protocol.js

```diff
@@ -6,16 +6,14 @@ module.exports = async function createHandler (options, session) {
     const appname = process.env.ssb_appname || options.appname || 'ssb'
 
     /** connect to running ssb-server */
-    if (!!options.ssbd === false) {
+    if (!!options.ssbd?.runServer === false) {
```

Comment:
> Would it also be possible to add some docs to the README about how this works?

I've changed this to connect to an already running sbot if ssbOptions.ssbd.runServer is false and documented it in the docs folder:

https://github.com/AgregoreWeb/agregore-browser/pull/196/commits/b2af81501c8b092aef08974ad3b28a51cb51fdc9#diff-f89bb456646ffb1a9491c3a422072f31b8320e29d00205ed4d3b7be04dbcd660R28